README.md

```diff
@@ -1191,6 +1192,14 @@ messagingSession.addObserver(observer);
 messagingSession.start();
 ```
 
+### Amazon Voice Focus and Echo Reduction
```

Comment:
This section is about the usage examples, say for example you want to provide a code snippet and then explain simply by it would go here.

For example:
```
### Voice Focus and Echo Reduction
```
**Use case 34.** Enabling Voice Focus ...

// Code snippet

**Use case 34.** Enable Voice Focus with Echo Reduction

// Code snippet
```

You can delete this as you detailed guide already.

Reply:
I agree with @devalevenkatesh. Use the guide as that is the more appropriate spot. All the feedback that we add here. Just incorporate that feedback in the guide.